InstallSpinnaker.sh

```diff
@@ -384,7 +404,7 @@ function add_apt_repositories() {
 }
 
 function install_java() {
-  if [ "$DOWNLOAD" != "true" ];then
+  if [[ $DOWNLOAD != true ]]; then
```

Comment:
Actually, turns out that it is "if $DOWNLOAD" without the brackets because 'false' is treated as a non-0 length string rather than a keyword or command returning a non-0 result.